components/admin_console/custom_url_schemes_setting.jsx

```diff
@@ -11,7 +11,7 @@ import LocalizedInput from 'components/localized_input/localized_input';
 
 import Setting from './setting';
 
-export default class CustomUrlSchemesSetting extends React.PureComponent {
+export default class CustomURLSchemesSetting extends React.PureComponent {
```

Comment:
I agree with updating the names of values that come from the server and their corresponding i18n keys and such, but I think we've tended towards the previous style of capitalizing acronyms in class names in the past. I don't think there's really an idiomatic style for naming in JavaScript though, so we could consider changing that here as well.

That all said, I think this change is fine. It is getting me thinking about us taking another crack at standardizing styling throughout the JS code base again though.